docs/tutorial/part-five/index.md

```diff
@@ -142,18 +142,20 @@ export const query = graphql`
 `
 ```
 
-The `console.log(data)` line is highlighted above. It's often helpful when
-creating a new component to console out the data you're getting from the GraphQL query
-so you can explore the data in your browser console while building the UI.
+`console.log(data)` रेखा के ऊपर प्रकाश डाला गया है। यह अक्सर मददगार होता है
```

Comment:
`रेखा के ऊपर प्रकाश डाला गया` => `लाइन ऊपर हाइलाइटेड`

Reply:
`हाइलाइटेड है।एक नया` => `हाइलाइटेड है। एक नया`

you missed a space before `एक`